parsigs/parse_sig_api.py

```diff
@@ -338,16 +337,11 @@ def _get_amount_from_frequency_tags(frequency):
             return 2
 
 
-def _get_times_from_latin(frequency):
-    latin_freq = _get_latin_frequency(frequency)
-    if latin_freq:
-        return latin_freq.times
-
-
-def _get_latin_frequency(frequency):
-    for latin_freq in latin_frequency_types.keys():
-        if latin_freq in frequency:
-            return latin_frequency_types[latin_freq]
+def _get_latin_frequency(frequency: str):
```

Comment:
this code can be improved by much, think how - where you want to save the same calculation hapenning over and over again. 
hint - ` stripped_freq = frequency.split()[0].replace('.', '')`

Reply:
yes an obvious fix, I moved that line out of the for. That is what you meant?